spring-kafka/src/main/java/org/springframework/kafka/support/KafkaStreamBrancher.java

```diff
@@ -86,6 +86,7 @@
 	 * @param stream {@code KStream} to split
 	 * @return the provided stream
 	 */
+	@SuppressWarnings("deprecation")
```

Comment:
DITTO

Reply:
```java
     * @deprecated since 2.8. Use {@link #split()} instead.
     */
    @Deprecated
    @SuppressWarnings("unchecked")
    KStream<K, V>[] branch(final Predicate<? super K, ? super V>... predicates);
```